be/src/common/configbase.cpp

```diff
@@ -214,6 +215,15 @@ bool Properties::load(const char* filename) {
         // compatible with doris_config
         key = std::regex_replace(key, doris_start, "");
 
+        if (key.compare("webserver_port") == 0) {
```

Comment:
To make it more extensible, we should declare the function name alias in `CONF_Int32`, instead of the property parsing code path.